### DISCUSSION TITLE
Your requirements could not be resolved to an installable set of packages.

### DISCUSSION THREAD

**Discussion by revenantreturns** [livewire/livewire#6067] [1. Help]
![image](https://github.com/livewire/livewire/assets/64325762/53e614b0-ca6e-4682-ab04-31f517be01c3)

I've tried a few things and it still doesn't work:

`composer install --ignore-platform-reqs`
or
`composer update --ignore-platform-reqs`

Laravel Framework 10.16.1
PHP 8.2.4 (cli) (built: Mar 14 2023 17:54:25) (ZTS Visual C++ 2019 x64)
Composer version 2.5.8 2023-06-09 17:13:21

**Comment by k-bon** [comment]
Have you tried putting the package in quotes? It seems some consoles in Windows get tripped up with "@" symbols.

`composer require "livewire/livewire:^3.0@beta"`

**Comment by joshhanley** [comment]
@revenantreturns thanks for reporting! There are some issues with installing the beta on windows, see this comment for more details and which command you should run https://github.com/livewire/livewire/pull/6076#issuecomment-1656336235

Hope this helps!
